Orange/preprocess/transformation.py

```diff
@@ -107,7 +107,10 @@ def __init__(self, variable, offset, factor):
         self.factor = factor
 
     def transform(self, c):
-        return (c - self.offset) * self.factor
+        if sp.issparse(c):
+            return c
```

Comment:
The above suggestion seems the right thing to do; however, it does not solve my problem —applying SVMs on sparse bag of words data — due to non-zero offsets.

Thinking a bit further, maybe my problem is not in _how_ we normalise, but _that_ we normalise. Should we even apply normalisation in this case? It only contains counts which in a sense already are in the same range (though this assumption breaks when there are also other, non BoW features present). Could we just turn normalisation off in this case? Though, I'm not sure how this could be elegantly achieved.